recipe/build-cpp.sh

```diff
@@ -2,101 +2,55 @@
 
 set -ex
 
-if [[ "${target_platform}" == osx* ]]; then
```

Comment:
I would keep this, unless it was removed from the conda-forge recipe. Basically, keep the scripts as close to conda-forge as possible.

Reply:
Addded back it again